roles/ceph-facts/tasks/facts.yml

```diff
@@ -327,6 +327,18 @@
   failed_when: false
   when: ceph_conf.stat.exists
 
+- name: get default crush rule value from running monitor ceph configuration
+  command: grep 'osd pool default crush rule' /etc/ceph/{{ cluster }}.conf
+  register: crush_rule_variable
+  changed_when: false
+  check_mode: no
+  failed_when: false
+  run_once: true
+  delegate_to: "{{ running_mon }}"
+  when:
+    - running_mon is defined
+    - not ceph_conf.stat.exists
```

Comment:
```suggestion
    - not ceph_conf.stat.exists | bool
```

Reply:
do you mind adding `| bool` on L328 as well?

Thanks!

Reply:
Good points. Thanks :)

Reply:
I know it was already like that before your patch but L328 is still missing the `| bool` if you could add it too that would be great :slightly_smiling_face:

Reply:
@guits I have added it. Sorry for missing it!